rotkehlchen/data/all_assets.json

```diff
@@ -4078,14 +4078,21 @@
         "type": "ethereum token"
     },
     "EDG": {
-        "coingecko": "edgeware",
+        "coingecko": "edgeless",
         "ethereum_address": "0x08711D3B02C8758F2FB3ab4e80228418a7F8e39c",
         "ethereum_token_decimals": 0,
         "name": "Edgeless",
         "started": 1487894400,
         "symbol": "EDG",
         "type": "ethereum token"
     },
+    "EDG-2": {
```

Comment:
you would also need to put a cryptocompare entry. Without one it will try to find "EDG-2" and fail. It's supported here:https://www.cryptocompare.com/coins/edgew/overview

So entry should be `"EDGEW"`